data/map planets.txt

```diff
@@ -3690,6 +3721,7 @@ planet Splashdown
 planet "Ssil Vida"
 	attributes emitter "remnant station" sheragi uninhabited
 	landscape land/station11
+	music ambient/machinery
 	description `The silence of this station rings in your ears, and as you explore you are increasingly surprised that it even has an atmosphere. It is obviously ancient, and patches of ambiguous green and purple material resembling fungus grow through the walls. Whoever built this station designed it for beings much larger than you. Either that, or they liked spacious environments. Even the shortest doorway you pass through is easily double your height. They are always wider than they are tall.`
```

Comment:
This one *definitely* shouldn't have sound, at least, not to begin with.

Reply:
Yeah I don't really know what I was thinking with this one.